dockerfiles/run/guice/provisioned/startup.sh

```diff
@@ -2,4 +2,4 @@
 
 wait-for-it.sh --host=localhost --port=9999 --strict --timeout=0 -- ./initialdata.sh &
 
-java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ -jar james-server.jar
\ No newline at end of file
+java -classpath '/root/james-server.jar:/root/james-server-jpa-guice.lib/*' -javaagent:/root/james-server-cli.lib/openjpa-2.2.1.jar -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ org.apache.james.JPAJamesServerMain
```

Comment:
Should investigate in the Manifest generation, it should work without classpath modification.